utils/motorcontroller/defaultconfig/NeoDefaults.java

```diff
@@ -0,0 +1,27 @@
+package frc.team670.mustanglib.utils.motorcontroller.defaultconfig;
+
+import frc.team670.mustanglib.utils.motorcontroller.MotorConfig.SparkMAXFactory.Config;
+
+public class NeoDefaults {
+    // for velocity control, all we need is frame 1, which has the velocity value
+
+    public Config VELOCITY_CONTROLLED_NEO = new Config();
+    VELOCITY_CONTROLLED_NEO.STATUS_FRAME_0_RATE_MS = 5000;
```

Comment:
If we're going to use either 5000 or 20 (which seems reasonable to me) in each of these, it's probably a little cleaner to define a constant above and then just use the constant in each of these places